### PR TITLE
zarik staging

### PR DESCRIPTION
**Pull request by zarik5**

### CODE REVIEW

File: alvr/server_openvr/LICENSE-Valve
Comment (The-personified-devil) [inline alvr/server_openvr/LICENSE-Valve:1]: Do we still have vendored openvr code? Otherwise this can go

Comment (zarik5) [reply alvr/server_openvr/LICENSE-Valve:1]: Yes. The C++ code was based on the VirtualDisplay sample code. the attribution header was stripped from some files. There are still `threadtools` and `d3drender` that still have it.

File: alvr/xtask/src/build.rs [hunk 1]
```diff
@@ -30,7 +30,6 @@ impl Display for Profile {
 pub fn build_server_lib(
```
Comment (The-personified-devil) [inline alvr/xtask/src/build.rs:30]: As far as I can tell this went from building a complete lib for SteamVR to kinda useless?

Comment (zarik5) [reply alvr/xtask/src/build.rs:30]: This could be used for Monado, if encoders and rendering are provided separately.

Comment (The-personified-devil) [reply alvr/xtask/src/build.rs:30]: I'd rather have a static lib in that case, but tbh the way it is currently is also how I've been working with monado in the past, so let's keep it and I can change it if I feel the need

**Comment by The-personified-devil** [comment]
You forgot to update the xtask formatting code

**Comment by zarik5** [comment]
Thanks, also fixed another occurrence in packaging.rs

**Review by The-personified-devil — APPROVED** [review]

**Pull request merged**